src/textual/message_pump.py

```diff
@@ -29,6 +32,32 @@ class MessagePumpClosed(Exception):
     pass
 
 
+T = TypeVar("T")
+
+
+def _is_coroutine_callable(call: Callable) -> bool:
+    if inspect.isroutine(call):
+        return inspect.iscoroutinefunction(call)
+    if inspect.isclass(call):
+        return False
+    call = getattr(call, "__call__", None)
+    return inspect.iscoroutinefunction(call)
+
+
+def _callable_in_thread_pool(call: Callable[..., T]) -> Callable[..., Awaitable[T]]:
+    async def inner(*args: Any, **kwargs: Any) -> T:
+        print(call)
```

Comment:
Is `print(call)` voluntary? It seems an artifact of a previous debug attempt.

Reply:
yep, mistake, will remove

Reply:
Removed. Thanks.